app/views/users/_form_fields.html.erb

```diff
@@ -41,6 +41,12 @@
   <%= f.select :organisation_id, organisation_options(f), organisation_select_options, { class: "chosen-select form-control", 'data-module' => 'chosen' } %>
 </p>
 
+<% if policy(User).flag_2sv? %>
+  <p class="form-group">
+    <%= f.check_box :requires_2sv %> <%= f.label :requires_2sv, "Requires 2SV" %>
```

Comment:
(It should be "2-step verification", with the hyphen.)

Is there a screenshot of how this looks?
Who is using this form and when are they using it?